gulpfile.coffee

```diff
@@ -9,6 +9,7 @@ execSyncOptions =
 buildAndTest = (done, includingParser = no) ->
 	try
 		execSync "clear; printf '\\033[3J'", execSyncOptions
+		execSync 'sync', execSyncOptions
```

Comment:
```suggestion
		execSync "clear; sync; printf '\\033[3J'", execSyncOptions
```

Reply:
With `&&` rather than `;` we will not to suffer with silence fail. 🙂
```suggestion
		execSync "clear && sync && printf '\\033[3J'", execSyncOptions
```